python/lsst/ts/observatory/control/data/mtcalsys_schema.yaml

```diff
@@ -159,6 +159,27 @@ properties:
     type: array
     items:
       type: number
+  constrained_random_exposure_times:
```

Comment:
Could we add `dacValue` to this configuration? Most of the bins will have different LED brightness, which correlated with dacValue